homeassistant/components/tuya/sensor.py

```diff
@@ -998,6 +999,24 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
             device_class=SensorDeviceClass.HUMIDITY,
             state_class=SensorStateClass.MEASUREMENT,
         ),
+        TuyaSensorEntityDescription(
+            key=DPCode.CUR_CURRENT,
+            name="Current",
```

Comment:
Please rebase the PR onto the latest dev and leverage the entity translations that is currently in the dev branch.